README.rst

```diff
@@ -295,6 +295,16 @@ For Fish, add this to ``~/.config/fish/completions/tmt.fish``::
 Open a new shell to enable completion. Or run the ``eval`` command
 directly in your current shell to enable it temporarily.
 
+This is however run every time you start a shell which can cause
+some delay. To speed it up, write the generated script to a file
+and then source it from your shell's configuration file. All
+of this can be achieved using ``tmt setup completion`` command.
+By default, it outputs the completion script to the terminal but
```

Comment:
Hm, I tried the non-install version of the command, but it does not work for me:

```
> tmt setup completion bash
No metadata found in the '.' directory. Use 'tmt init' to get started.
```

When under an `fmf` directory it lists available objects just as plain `tmt` would do. Any idea what might be wrong?

Reply:
Hm, this seems to be caused by the fact that the `setup` command is under `main` group which tries to load the tree. But the objects should not be listed (they should only be there if invoked subcommand is None). I find it a bit weird that this happened to you. I tried locally running it in an fmf directory and outside fmf directory and both worked fine. But nevertheless, there is no point in loading the tree in this case, I will see if there is a better way to structure this CLI.

Reply:
Hm, weird, I'm not able to reproduce the failure now. Seems the time has fixed problem ;-)